python/opdb/models.py

```diff
@@ -453,6 +453,7 @@ class pfs_design(Base):
     is_obsolete = Column(Boolean)
 
     tiles = relation(tile, backref=backref('pfs_design'))
+    pfs_design_agcs = relation('pfs_design_agc', back_populates='pfs_design')
 
```

Comment:
```suggestion
pfs_design_fibers = relation('pfs_design_fiber', back_populates='pfs_design')

```

Reply:
Please add this in the same reason.
Also could you add the following line in pfs_design_fiber class?
`pfs_design = relation('pfs_design', back_populates='pfs_design_fibers')`

Reply:
I added the relation.